src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -587,3 +587,100 @@ file that was distributed with this source code.
         });
     </script>
 {% endblock %}
+
+{% block sonata_type_model_list_widget %}
```

Comment:
I've copied this from https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/29aeea8ba292ac3f626c56e78484b2c10b04cbe2/src/Resources/views/Form/form_admin_fields.html.twig#L56 since the code looks persistence agnostic and is more updated than mongodb one and we can remove it from the persistence bundles.